content/en/docs/tutorials/stateful-application/zookeeper.md

```diff
@@ -26,7 +26,7 @@ Kubernetes concepts:
 - [Pods](/docs/concepts/workloads/pods/)
 - [Cluster DNS](/docs/concepts/services-networking/dns-pod-service/)
 - [Headless Services](/docs/concepts/services-networking/service/#headless-services)
-- [PersistentVolumes](/docs/concepts/storage/volumes/)
+- [PersistentVolumes](/docs/concepts/storage/persistent-volumes/) 
```

Comment:
Thank you for updateing the link! 
Very minor nit – there's a trailing white space added, but overall, LGTM!
/lgtm

[Preview Page](https://deploy-preview-44402--kubernetes-io-main-staging.netlify.app/docs/tutorials/stateful-application/zookeeper/)





```suggestion
- [PersistentVolumes](/docs/concepts/storage/persistent-volumes/)
```